### DISCUSSION TITLE
How to make all instances of a specific block breakable even through Grief Prevention?

### DISCUSSION THREAD

**Discussion by NanoAi** [GriefPrevention/GriefPrevention#1251] [Q&A] [answered]
How would I allow a specific block to be able to be broken regardless of if it's in a claim, of if there's a Siege happening? I have a mod that drops all of a players items into a head when they die. So I need those heads to be breakable no matter what.

**Comment by RoboMWM** [comment, marked as answer]
Hmm, probably easiest to just have an addon that uncancels if that's the block intended to be broken. Would allow for further tuning, since I'm sure you wouldn't want regular skulls to be broken.

**Reply by NanoAi** [reply]
I won't be able to test it out until Monday, as a quick "for now" fix is there a way to just whitelist PLAYER_HEAD and PLAYER_WALL_HEAD? (Players can't place them as blocks any way, only in items frames/armour stands.)

**Reply by RoboMWM** [reply]
don't know if there's any way via the config, but a simple plugin can uncancel BlockBreakEvent if event#getBlock is equal to either of those two enums.
